.github/workflows/sync-critical-path.yml

```diff
@@ -26,6 +26,12 @@ jobs:
           java-version: '17'
           distribution: 'adopt'
 
+      - name: Create test account for Sync and return the recovery code
+        uses: duckduckgo/sync_crypto/action@main
+        id: sync-recovery-code
+        with:
+          debug: true
```

Comment:
Here's a suggestion. You could add a `no-teardown: 'false'`, just as a reminder that there's a teardown happening by default. Next time someone tries to debug a broken build, that will be a detail worth remembering.

Reply:
but do we need to always run the workflow with `debug: true`? or is that only for debug purposes?

Reply:
We don’t need debug true, but in case something goes wrong it’s easier to read. I like @jpbochi idea, adding it.